fifteen_min/src/find_home.rs

```diff
@@ -35,7 +35,8 @@ impl FindHome {
             )
             .flex_wrap(ctx, Percent::int(50)),
             ctx.style()
-                .btn_solid_primary_text("Search")
```

Comment:
This is representative of the mechanical changes that make up 95% of the lines in this PR.